src/test/java/com/coherentsolutions/training/aqa/java/web/nikonova/utils/FailedTestListener.java

```diff
@@ -0,0 +1,53 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.utils;
+
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersSettings;
+import io.qameta.allure.Allure;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+import org.testng.IInvokedMethod;
+import org.testng.IInvokedMethodListener;
+import org.testng.ITestContext;
+import org.testng.ITestResult;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static com.coherentsolutions.training.aqa.java.web.nikonova.objects.BaseTest.log;
+
+public class FailedTestListener implements IInvokedMethodListener {
+
+    public static String screenshotsFolderName;
```

Comment:
Please remove this field from the class and create a String variable in the method and use it